src/main/ui/app/route/Show.jsx

```diff
@@ -74,7 +76,7 @@ export default class Show extends Component {
             <Label bsStyle={statusClass}>{status}</Label>
             {addon.tags ?
              addon.tags.map(t =>
-                 <Label bsStyle="default">{t}</Label>
+                 <Link to={`/search?&tag=${t}`}><Label bsStyle="default">{t}</Label></Link>
```

Comment:
With this a user can find one module, and then browse to other modules with the same tag. But we also want to give them a page that lists all the tags and lets them click on one. (Fine to do that as part of a different ticket, so we don't get further derailed from our originally scheduled work this week.)